shell/detail/fleet.cattle.io.gitrepo.vue

```diff
@@ -125,6 +142,21 @@ export default {
       class="mt-20"
       :need-related="false"
     >
+      <Tab
```

Comment:
Is there an issue to track the addition of this tab? It seems outside the scope of #9866. Semi-related, the table formatting looks off to me: 
![Screenshot 2024-05-16 at 9 51 45 AM](https://github.com/rancher/dashboard/assets/42977925/f617294f-1114-43ad-b2da-e2e437bb4ae9)

Should I file a separate issue for that?

Reply:
So, all of this comes from this PR: https://github.com/rancher/dashboard/pull/10776 which is tied to https://github.com/rancher/dashboard/issues/10339. I found some problems within Fleet regarding counts, interfaces throwing errors and the code seemed a bit too much so I went ahead and added the improvements that I felt were needed.

You're right, there's something off when looking at that table in terms of formatting. Never payed attention to that... 🤔  good to have another set of 👀  🙏 

I am going to remove this tab from this PR... I quite out of scope. Also, thinking about it, RBAC might also limit the clusters a user can "see" so I don't think it's needed right now. I can create an issue to track it and maybe we add it in the future.

Thanks for the review!